endaq/plot/plots.py

```diff
@@ -242,35 +240,43 @@ def get_pure_numpy_2d_pca(df: pd.DataFrame,
     return fig
 
 
-def gen_map(df_map: pd.DataFrame, mapbox_access_token: str, filter_points_by_positive_groud_speed: bool = True,
-            color_by_column: str = "GNSS Speed: Ground Speed") -> go.Figure:
+def gen_map(df_map: pd.DataFrame, mapbox_access_token: str, filter_positive_color_vals: bool = True,
+            lat: str = "Latitude",
+            lon: str = "Longitude",
```

Comment:
Unless it really is arbitrary, we should probably use the `Literal` type annotation.

Reply:
I've seen `Literal` used before in our code when there are a few different discrete options like:
```
scaling: typing.Literal[None, "density", "spectrum", "parseval"] = None
```

But for this the variable is defining a column name in a user defined dataframe so it could be any arbitrary name. They may label it as `"Lat"` or `"GNSS: Latitude"` or even `"A"`

But I may be completely misunderstanding what `Literal` is for so let me know if you think this should still change in light of my explanation!